### DISCUSSION TITLE
Why isnt elastalert2 using the latest elasticsearch version (7.15)?

### DISCUSSION THREAD

**Discussion by LennartssonAlex** [jertel/elastalert2#532] [Q&A] [answered]
We moved from elastalert to elastalert2 in one of our projects and i saw that i needed to downgrade my elasticsearch version. The requirements file specifies version 7.0.0, before moving to elastalert2 we used 7.10.

Why isnt elastalert2 using the latest version?

**Comment by nsano-rururu** [comment]
Do you mean elasticsearch-py?

**Comment by nsano-rururu** [comment, marked as answer]
This is because some functions do not work properly.
https://github.com/jertel/elastalert2/pull/90

**Reply by nsano-rururu** [reply]
It has been confirmed that elsticsearch-py 7.0.0 works by connecting to the latest elsticsearch 7.15.

**Reply by nsano-rururu** [reply]
If you set elsticsearch-py to 7.15, you will not be able to connect elsticsearch-oss and opensearch except for some features that do not work.

**Reply by LennartssonAlex** [reply]
Yes i ment elasticsearch-py, thanks for your answer.
